tests/python/integration/test_meta_schedule_auto_tensorize.py

```diff
@@ -393,7 +394,7 @@ def convert_layout(mod):
             )
         print(profiler.table())
 
-        # Compile without meta-scheduler for correctness check
+        # Compile without MetaScheduler for correctness check
```

Comment:
This is "MetaSchedule", not "MetaScheduler"

```suggestion
        # Compile without MetaSchedule for correctness check
```